cmd/wksctl/profile/enable/enable.go

```diff
@@ -87,3 +96,49 @@ func profileEnableRun(params profileEnableFlags) error {
 
 	return nil
 }
+
+func doIgnoreFiles(profilePath string) error {
+	ignoreFilePath := path.Join(profilePath, constants.WKSctlIgnoreFilename)
+	if _, err := os.Stat(ignoreFilePath); err == nil {
+		log.Infof("Ignoring files declared in %s...", constants.WKSctlIgnoreFilename)
+		file, err := os.Open(ignoreFilePath)
+		if err != nil {
+			return err
+		}
+		defer file.Close()
+		pathsToIgnores, err := parseDotIgnorefile(profilePath, file)
+		if err != nil {
+			return err
+		}
+		if err := removePathsFromGit(pathsToIgnores...); err != nil {
+			return err
+		}
+		log.Info("Ignored files successfully.")
+	}
+	return nil
+}
+
+func parseDotIgnorefile(dir string, reader io.Reader) ([]string, error) {
+	result := []string{}
+	scanner := bufio.NewScanner(reader)
+	re := regexp.MustCompile(`(?ms)^\s*(?P<pathToIgnore>[^\s#]+).*$`)
+	for scanner.Scan() {
+		groups := re.FindStringSubmatch(scanner.Text())
+		if len(groups) != 2 {
+			continue
+		}
+		pathToIgnore := groups[1]
+		result = append(result, path.Join(dir, pathToIgnore))
+	}
+	return result, nil
+}
+
+func removePathsFromGit(paths ...string) error {
+	if err := git.RmRecursive(paths...); err != nil {
+		return err
+	}
+	if err := git.Commit("Ignored files declared in .wksctlignore"); err != nil {
```

Comment:
Probably should use `constants.WKSctlIgnoreFilename` instead of hardcoded `.wksctlignore`

Reply:
nice catch @fbarl thank you!